packages/app/utils/datetime.ts

```diff
@@ -86,3 +81,16 @@ export const formatFrequencyHours = (hours: number) => {
 //eg.  "1 day" | "4 days"
 const frequencyString = ({ name, number }: { name: string; number: number }) =>
   `${number} ${number === 1 ? name : name.concat("s")}`;
+
+// from date
```

Comment:
Also this one. I can't understand the purpose of this comment. Should we split time and date related logic into different files?

Reply:
The others are from timestamp, this one is from a `Date`. That's the difference.

Reply:
I think I can remove the comments as the name of the function and args are very descriptive.

Reply:
Looks good to me!